### PR TITLE
Adding todo list

### PR DESCRIPTION
**Pull request by yahyaelganyni1**
this is the logic and style for the todo list 

- we have an HTML file with the main requirement  from ul and heading and input tag
- the CSS we have styled all the elements
- in the javascript, I have added  an array of objects with the three requirement

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (mikenath223) [inline .gitignore:1]: - [x] You seem to have committed node_modules, kindly avoid committing it. Please ensure node_modules isn't part of the committed files.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+# HTML & CSS3 & JavaScript Course
+
+If you are not familiar with linters, read [root level README](../README.md).
+
+## Set-up GitHub Actions
+
+Please do the following **steps in this order**:
```
Comment (mikenath223) [inline README.md]: - [x] This readme doesn't describe your project. Please use this [readme template](https://github.com/microverseinc/readme-template) and ensure you customise it to your project.

File: SRC/index.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    description: 'do stuff 3',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'do stuff 4',
+    completed: false,
+    index: 3,
+  },
+];
```
Comment (mikenath223) [inline SRC/index.js]: - [ ]  On loading `index.html`, this list of todos should show up on the page. Please fix it.

File: SRC/index.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Todo List</title>
+  </head>
+  <body>
+    <div class="main-div">
+      <input type="text" class="ibput-box" />
+      <h2>today todos</h2>
+      <ul class="the-list"></ul>
+      <button class="clear-btn">Clear all completed</button>
+    </div>
```
Comment (mikenath223) [inline SRC/index.html]: - [ ] Your project should be styled to look exactly like the given design. Kindly ensure you update the styling.

| Original | Yours
| ------- | ---------- |
|  |  |

File: SRC/index.js (hunk 3 above)
Comment (yahyaelganyni1) [reply SRC/index.js]: I don't understand this one please explain to me because on my end it's working ok

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @yahyaelganyni1🖐🖐👏👏👏** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- Please add more description to your readme file.
- Please add a live demo to your Todo list project.
- Your project is not functional, please make the change to solve the issue. I can't see the input field to add the list.
- Your project [see screenshot](https://user-images.githubusercontent.com/61562626/125595117-0a4b48e5-1da9-46db-a970-b8e0e1158076.png), and the wanted output =>[Capture d’écran](https://user-images.githubusercontent.com/61562626/125595356-dadcf2e1-15cc-4e09-b08d-9b0e86a7be61.png)👈




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 5]
```diff
@@ -7,3 +7,11 @@ this is a todo list with javascript
 ### desktop
 
 ![desktop](./img/website-img.png)
+
```
Comment (dasileker) [inline README.md]: - [x] please add more details to your readme file and a live demo link

Comment (yahyaelganyni1) [reply README.md]: hello Zerradi 

did you try to open it from webpack server ? because it's working on my pc

File: README.md (hunk 2 above)
Comment (bellom) [reply README.md]: @yahyaelganyni1 
The first and second reviewers explicitly told you to work on your README file, in fact, you were given the [template](https://github.com/microverseinc/readme-template) up there 👆 .

KIndly follow the template and update your README file.

Your REAMDE file:
image

**Comment by bellom** [comment]
@yahyaelganyni1 

I saw your note on the CRR, saying the reviewer should message you on slack if there is an issue, I'm sorry I might not be able to message you on slack, we need to keep every conversation in here so that it will be easier to make corrections and also, to improve the Microverse system.

But I drop you a message on slack thou 🙂

**Comment by bellom** [comment]
@yahyaelganyni1 while waiting for your to reply in here, I saw you pushed some changes but still doesn't look professional, see here 👇 
image


But kindly update this in the next milestone.

**Review by bellom — APPROVED** [review]
## Hi @yahyaelganyni1  👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**